config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "users/:user_id/posts", to: "posts#index", as: "user_posts"
+  get "users/user_id/posts/:id", to: "posts#show", as: "user_post"
+  get "users", to: "users#index" 
+  get "users/:id", to: "users#show", as: "user"
 end
```

Comment:
## Not a requirement
-  Job well done here, this way you will have custom URLs. Another simplest way to have your routes set up is when you use the `resources` keyword👏👏

Reply:
Hello good day, this is not a requirement, but you still invalidated my project and requested for changes?